src/profile/travelocity/extension/tab-type.js

```diff
@@ -0,0 +1,11 @@
+(function() {
+    if (localStorage.getItem(window.location.href)) {
```

Comment:
window.location.href contains all the query parameters, too. e.g. "https://www.google.com/search?q=window.location.href&oq=window.location.href+&aqs=chrome..69i57j0l5.3951j0j7&sourceid=chrome&ie=UTF-8"

Do you want to include all of that or just up to the first questions mark?